scrapi/harvesters/__init__.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+_, __all__, files = next(os.walk(os.path.dirname(__file__)))
+
+for name in files:
+    if name[-3:] == '.py' and name != '__init__.py':
+        __all__.append(name[:-3])
+
```

Comment:
And not in a bad way, though maybe add some commentary, since there's a good chance someone could be completely mystified by this while writing a harvester.